apps/web/src/components/create-project/create-project-form.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import type { Upload } from "@mux/mux-node/resources/video/uploads.mjs";
+import MuxUploader from "@mux/mux-uploader-react";
+import { useMutation } from "@tanstack/react-query";
+import { useSession } from "next-auth/react";
+import { useRouter } from "next/navigation";
+
+import type { Organization } from "~/lib/validators/organization";
+import type { InsertProject, Project } from "~/lib/validators/project";
+import { createProject, deleteProjectById } from "~/server/actions/project";
+import { createVideoEntry } from "~/server/actions/videoEntry";
+import type { InsertVideoEntry, VideoEntry } from "~/lib/validators/videoEntry";
+
+type VideUploadFormProps = {
+  upload: Upload;
+  org: Organization;
+};
+
+export default function VideUploadForm({ upload, org }: VideUploadFormProps) {
+  const session = useSession();
+  const router = useRouter();
+
+  const { data: newProject, mutate: createProjectMutation } = useMutation<
+    Project | null,
+    Error,
+    InsertProject
+  >({
+    mutationKey: ["create-project"],
+    mutationFn: async (data) => {
+      const [newProject, err] = await createProject(data);
+
+      if (err !== null) {
+        console.error(err);
+      }
+
+      return newProject;
+    },
+  });
+  
+  console.log(newProject);
+
+  const { mutate: createVideoEntryMutation } = useMutation<
+    VideoEntry | null,
+    Error,
+    InsertVideoEntry
+  >({
+    mutationKey: ["create-video-entry"],
+    mutationFn: async (data) => {
+      const [newEntry, err] = await createVideoEntry(data);
+
+      if (err !== null) {
+        console.error(err);
+      } else {
+        createProjectMutation({
```

Comment:
also toast the creation

Reply:
It is unnecessary since the user'll be redirected on a sucessful project cretion.